envoy/stats/store.h

```diff
@@ -81,6 +158,50 @@ class Store : public Scope {
   virtual void forEachSinkedCounter(SizeFn f_size, StatFn<Counter> f_stat) const PURE;
```

Comment:
it took me a while to understand the semantics of the two parameters, I know this is not your fault, but it would be nice if we can add some comments about the parameters.

Reply:
the doxygen comments are above. Do you have a clarification for those?

Reply:
sorry, it's folded so I didn't see the parameters. All set.